### PR TITLE
set width boundaries for long usernames in activity center 

### LINKED ISSUE #13178: Long username is overlaid on timestamp in Activity center (Android)
# Bug Report

## Problem 
[comment]: # (A problem description)

photo_2022-03-17_12-09-46

### Reproduction

1. UserA with a long username creates a new 1-1 chat with UserB
2. UserB (Android) opens Home screen > navigates to the Activity center (the bell at the top right)

### Additional Information

- Status version: nightly 17/03
- Operating System: Android

[comment]: # (Please, add logs/notes if necessary)
cc @siddarthkay

### PR DESCRIPTION
**Pull request by siddarthkay**
[comment]: Fixes https://github.com/status-im/status-react/issues/13178

Platforms
Android
iOS
status: ready

### CODE REVIEW

File: src/status_im/ui/screens/notifications_center/styles.cljs [hunk 1]
```diff
@@ -68,9 +69,11 @@
    :left     16})
 
 (def title-text
-  {:margin-left  72
-   :margin-top   12
-   :margin-right 50})
+  (let [title-text-width (* @(re-frame/subscribe [:dimensions/window-width]) 0.65)]
```
Comment (flexsurfer) [inline src/status_im/ui/screens/notifications_center/styles.cljs]: its better to pass width as a parameter from the view

**Review by flexsurfer — CHANGES_REQUESTED** [review]

File: src/status_im/ui/screens/notifications_center/styles.cljs (hunk 1 above)
Comment (siddarthkay) [reply src/status_im/ui/screens/notifications_center/styles.cljs]: I agree! 
working on this

Comment (siddarthkay) [reply src/status_im/ui/screens/notifications_center/styles.cljs]: @flexsurfer : done

File: src/status_im/ui/screens/notifications_center/styles.cljs [hunk 2]
```diff
@@ -1,6 +1,7 @@
 (ns status-im.ui.screens.notifications-center.styles
   (:require [quo.design-system.colors :as colors]
-            [quo.design-system.spacing :as spacing]))
+            [quo.design-system.spacing :as spacing]
+            [re-frame.core :as re-frame]))
```
Comment (flexsurfer) [inline src/status_im/ui/screens/notifications_center/styles.cljs]: re-frame not used anymore

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/ui/screens/notifications_center/styles.cljs (hunk 2 above)
Comment (siddarthkay) [reply src/status_im/ui/screens/notifications_center/styles.cljs]: fixing and squashing asap :)

Comment (flexsurfer) [reply src/status_im/ui/screens/notifications_center/styles.cljs]: thanks :) no rush :)

**Comment by pavloburykh** [comment]
@siddarthkay thanx for the PR. Please, take a look at this issue

### ISSUE 1 

New usernames and messages are highlighted in activity center. 

STEPS: 

1. Generate new chat messages in activity center by user with short and long username
2. Pay attention at messages in Activity center

Actual result: 
1. In case of message from user with long username - both username, message and timestamp are highlighted in Activity center
2. In case of message from user with short username - only username is highlighted in Activity center

OS: Android/IOS

photo_2022-03-21 20 22 20


### ISSUE 2

Please, take a look at lint errors in last builds 
https://ci.status.im/blue/organizations/jenkins/status-react%2Fprs%2Fandroid/detail/PR-13185/3/pipeline

**Comment by siddarthkay** [comment]
@pavloburykh : Um yeah my bad, I left in the background highlighting( which i use internally for debugging ) , I'm pushing a fix for that

File: src/status_im/ui/screens/notifications_center/styles.cljs (hunk 2 above)
Comment (siddarthkay) [reply src/status_im/ui/screens/notifications_center/styles.cljs]: Done :)

**Comment by siddarthkay** [comment]
@pavloburykh : fix pushed, should be good for QA now.

File: src/status_im/ui/screens/notifications_center/styles.cljs (hunk 2 above)
Comment (siddarthkay) [reply src/status_im/ui/screens/notifications_center/styles.cljs]: removed unused import

**Comment by pavloburykh** [comment]
@siddarthkay thanx for fixes! 

### ISSUE 3
It seems that on IOS (iPhone X) long username is still located to close to timestamp. That's definitely a minor but if this is an easy fix lets improve it. 

Below 2 screenshots for comparison from IOS (iPhone X) and Android (Samsung Galaxy A52) with a message from same user 

iPhone x
photo_2022-03-22 10 20 39


Samsung Galaxy A52
photo_2022-03-22 10 21 34

**Review by briansztamfater — APPROVED** [review]

**Comment by siddarthkay** [comment]
@pavloburykh : I'm working on reproducing this in iphoneX Simulator and then will push a fix

**Comment by pavloburykh** [comment]
> @pavloburykh : I'm working on reproducing this in iphoneX Simulator and then will push a fix

@siddarthkay thank you! let me know if you need any help on reproduction.

**Comment by siddarthkay** [comment]
Latest fix should take care of this case @pavloburykh 
Screenshot on iPhone X below 
Simulator Screen Shot - iPhone X - 2022-03-23 at 23 50 18

**Comment by pavloburykh** [comment]
@siddarthkay thanx for fixes, ready for merge. Sorry for testing delay, was busy by release testing.

**Pull request merged**